README.md

```diff
@@ -111,11 +119,10 @@ lifecycleScope.launchWhenStarted {
 }
 ```
 
-For convenience, there are `Store.get(key)`, `Store.stream(key)` and `Store.fresh(key)` extension functions.
+For convenience, there are `Store.get(key)` and `Store.fresh(key)` extension functions.
 
-* `suspend fun Store.get(key: Key): Value`: This method returns a single value for the given key. If available, it will be returned from the in memory cache or the persister.
-* `suspend fun Store.fresh(key: Key): Value`: This method returns a single value for the given key that is obtained by querying the fetcher.
-* `suspend fun Store.stream(key: Key): Flow<Value>`: This method returns a `Flow` of the values for the given `key`.
+* `suspend fun Store.get(key: Key): Value`: This method returns a single value for the given key. If available, it will be returned from the in memory cache or the persister. An error will be thrown if not value is available in either the `cache` or `persister`, and the `fetcher` fails to load the data from the network.
```

Comment:
```suggestion
* `suspend fun Store.get(key: Key): Value`: This method returns a single value for the given key. If available, it will be returned from the in memory cache or the persister. An error will be thrown if no value is available in either the `cache` or `persister`, and the `fetcher` fails to load the data from the network.
```